.github/workflows/beep-test.yml

```diff
@@ -0,0 +1,40 @@
+name: Testing - main
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    strategy:
+      max-parallel: 20
+      matrix:
+        os: [
+          ubuntu-latest,
+          macos-latest,
+          windows-latest
+        ]
+        python-version: [3.6, 3.7, 3.8]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install -e .[tests]
+    - name: pytest
+      env:
+        MPLBACKEND: "Agg"
+        BEEP_ENV: "dev"
+      run: |
+        pytest beep --color=yes --cov=camd --cov-report html:coverage_reports
```

Comment:
Should this be: `pytest beep --color=yes --cov=beep --cov-report html:coverage_reports`?